### ISSUE TITLE
Seek and pause on chrome restart player

### ISSUE THREAD

**Issue by Riiuku** [goldfire/howler.js#1289]
Hello, when I am using chrome and want to change seek or pause currently playing song, my player restarting it, This occurs only on Chrome. On Firefox for example everything working well. Somone had similar problem and know how to resolve it? I don't want to change html5 to false because I want to stream song, not wait for full dowloading :(

**Comment by mxvin** [comment]
You're right. Demo player with `html5: true` is breaking after seeking or pausing player, Toggling `html5: false`, make seek and pause behave correctly on Chrome. idk what possible workaround for this problem

**Comment by mxvin** [comment]
Nah, this is just http header issue. see here https://stackoverflow.com/questions/37044064/html-audio-cant-set-currenttime#comment97935607_52520120
You need to set header "Accept-Ranges: bytes" on the audio file. Chrome seems more sensitive about this compare to firefox for assigning the audio.currentTime

**Comment by tettusud** [comment]
@mxvin could you help me how can I work around this? I am facing same issue when html5 is true, its resetting to start.
